### DISCUSSION TITLE
Nvitop equivalent

### DISCUSSION THREAD

**Discussion by davidezari** [ROCm/TheRock#8014] [General]
Searching a way to monitor in realtime the load of the GPU, on NV side there is this nice tiny tool communicating with smi and getting an running update about what is going on.

Looking on ROCm side I found only 3-4 years old repos like these one
https://github.com/ai-z/ai-z
https://github.com/avimanyu786/ROCm-SMI-Real-Time
That I dont even know if are working on last release.
Wanted to ask if I'm missing something and there is another tool an end user can rely on, or previously one are the solutions commonly used, so that there was no need for something new like nvitop.

**Comment by briansp2020** [comment]
https://github.com/Umio-Yasuno/amdgpu_top

**Comment by farshadghodsian** [comment]
If you are still looking for a tool **nvtop** is my go to. It supports both NVidia and AMD GPUs and has a cool interface: https://github.com/Syllo/nvtop

**Comment by nrhevu** [comment]
You can try this: https://github.com/nrhevu/roctop
